### PR TITLE
ADAPT-3983: Remove 403/404 paths as they are now redirects. (#233)

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Cherry picks commit 412c340ef0bce9884baa3bb8d595552b5b4b413f out of dev for deployment

# Review By (Date)
- Today would be nice

# Criticality
- Medium / High

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "saa-alumni",
-      "version": "0.1.1",
+      "version": "0.1.2",
```
Comment (yvonnetangsu) [inline package-lock.json:9]: @sherakama Is this behind?

**Review by yvonnetangsu — APPROVED** [review]
Thanks for this! The changes look good to me so preapproving. Got a question about the version though 🤔

File: package-lock.json (hunk 1 above)
Comment (sherakama) [reply package-lock.json:9]: hmm perhaps.

Comment (sherakama) [reply package-lock.json:9]: Ah, that's just package-lock. This is fine.

Comment (yvonnetangsu) [reply package-lock.json:9]: Cool - all GTG then! 🎉

**Review by mattanglin — APPROVED** [review]
lgtm 👍🏼 . I assume the redirects are being handled at our DNS level now?

**Comment by sherakama** [comment]
> lgtm 👍🏼 . I assume the redirects are being handled at our DNS level now?

It is a mix. We have redirects from storyblok and custom 404/403 pages set up there now that this code is overriding.

**Pull request merged**